pkg/dockerutil/dockerutils.go

```diff
@@ -852,12 +852,13 @@ func GetHostDockerInternalIP() (string, error) {
 	return hostDockerInternal, nil
 }
 
-// RemoveImage removes an image
+// RemoveImage removes an image with force
 func RemoveImage(tag string) error {
 	client := GetDockerClient()
-	err := client.RemoveImage(tag)
+	err := client.RemoveImageExtended(tag, docker.RemoveImageOptions{Force: true})
+
 	if err == nil {
-		util.Success("Deleting docker image %s", tag)
+		util.Success("Deleted docker image %s", tag)
```

Comment:
Unrelated changes?

Reply:
It is unrelated. Was in the neighborhood and did no harm.